server/fleet/mdm.go

```diff
@@ -5,6 +5,17 @@ import (
 	"time"
 )
 
+type MDMAppleProfileStatus string
```

Comment:
Do we want to relocate to `fleet/apple_mdm` (or vice-versa)?

Reply:
great catch! thanks!